api/pom.xml

```diff
@@ -301,6 +301,12 @@
             <version>1.11.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
We should already have AssertJ available in the dependency tree. If we need to upgrade, then we probably need to upgrade Spring starter parent first.

Reply:
Removed